templates/analytics/analytics-deployment.yaml

```diff
@@ -65,4 +65,8 @@ spec:
       imagePullSecrets:
       - name: {{ $webapp.registry_secret }}
       {{- end }}
+      {{- if .Values.hostAliases }}
+      hostAliases:
+        {{- .Values.hostAliases  | toYaml | nindent 6  }}
```

Comment:
`nindent 8` to stay coherent with the other modifications ?

Reply:
done